js/tji-datasets-explore.js

```diff
@@ -224,19 +230,50 @@ TJIChartView.prototype.get_data = function() {
   var url = '/cdr_compressed.json';
   jQuery.getJSON(url)
     .done(function(data){
-      jQuery(that.charts_elt_id).empty();
+      // Prepare the data
       that.state.data = data;
       that.decompress_data();
       that.transform_data();
+      that.state.filtered_data = that.state.data;
+
+      // Prepare HTML contents
+      jQuery(that.charts_elt_id).empty();
+      that.create_above_charts_area();
       that.create_charts();
       that.create_filter_panel();
       that.attach_events();
+      that.get_complete_data();
     })
     .fail(function(jqxhr, textStatus, error){
       console.log('error fetching data from: ' + url, error);
     })
 }
 
+
+TJIChartView.prototype.get_complete_data = function() {
+  var that = this;
+  var url = '/cdr_full.csv';
```

Comment:
can you make url for the complete data_set and the compressed data_set params so this can be flexible for multiple data sets? No worries, if not, I can update this when I get to loading the police shootings data.